.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,11 @@
-<!--
+## PR Checklist
```

Comment:
The Problem investigation and results should be in the issue.  

This checklist is all items that by our defined process is supposed to be done by the contributor except for the one new item this PR specifically adds and the one @markekraus  asked me to add.

Change Summary is supposed to be what you call the "fix description" and we the link in the checklist describes how to produce this.  Change Summary is more appropriate because not all PRs are fixes.

Reply:
> The Problem investigation and results should be in the issue.

I agree, but in fact, we see another. The initial description in a Issue is almost always different from what we need to do in PR. Moreover, in most cases Issues do not contain a final opinion.

+1 for Change Summary.

Non-blocking comment.